gpu-enabler/src/main/scala/com/ibm/gpuenabler/CUDAManager.scala

```diff
@@ -132,6 +132,31 @@ private[gpuenabler] class CUDAManager {
   private[gpuenabler] def freeGPUMemory(ptr: Pointer) {
     JCuda.cudaFree(ptr)
   }
+  
+  private[gpuenabler] def unCacheGPULRU(sz: Int) = {
+    /* remove cached RDDs on GPU automatically
+     * without ignoring user's cache (unCache) indication.
+     * Since spark can't estimate RDD size via distributed data
+     * on eacn nodes, this method removes cached RDDs on GPU
+     * in the partitions units.
+     * this method is called on allocateGPUMemory.
+     * In order to reuse cached RDDs on gpu as much as possible,
+     * adopt LRU as a cache strategy.
+     *
+     * gpuMemThreshold is a threshold for sadety of a GPU memory.
+     * when this variable has "1000000000",
+     * if GPU free space become less than "1000000000B (1 GB)",
+     * auto cached RDD on GPU is evicted on a LRU policy.
+     */
+    val gpuMemThreshold = 1000000000 //1GB
```

Comment:
Can we make this value `1000000000` configurable? For example, read this value from a property or option.

Reply:
Yes you can.
If necessary, I will add that property (or option).